docs/eventing/samples/kafka/kafka_setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+# Turn colors in this script off by setting the NO_COLOR variable in your
+# environment to any value:
+#
+# $ NO_COLOR=1 test.sh
+NO_COLOR=${NO_COLOR:-""}
+if [ -z "$NO_COLOR" ]; then
+  header=$'\e[1;33m'
+  reset=$'\e[0m'
+else
+  header=''
+  reset=''
+fi
+strimzi_version=`curl https://github.com/strimzi/strimzi-kafka-operator/releases/latest |  awk -F 'tag/' '{print $2}' | awk -F '"' '{print $1}' 2>/dev/null`
+function header_text {
+  echo "$header$*$reset"
+}
+header_text "Using Strimzi Version:                  ${strimzi_version}"
+header_text "Strimzi install"
+kubectl create namespace kafka
+curl -L "https://github.com/strimzi/strimzi-kafka-operator/releases/download/${strimzi_version}/strimzi-cluster-operator-${strimzi_version}.yaml" \
+  | sed 's/namespace: .*/namespace: kafka/' \
+  | kubectl -n kafka apply -f -
+header_text "Applying Strimzi Cluster file"
+kubectl -n kafka apply -f "https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/${strimzi_version}/examples/kafka/kafka-persistent-single.yaml"
+header_text "Waiting for Strimzi to become ready"
+sleep 5; while echo && kubectl get pods -n kafka | grep -v -E "(Running|Completed|STATUS)"; do sleep 5; done
```

Comment:
Not a super big deal, but maybe we can remove these 2 lines.
The loop is prematurely ended because the operator becomes in running state but it hasn't created any other pods yet.

Perhaps we increase the initial sleep to something like 30 seconds.